Source/WebCore/rendering/RenderTheme.cpp

```diff
@@ -1559,7 +1559,7 @@ Color RenderTheme::systemColor(CSSValueID cssValueId, OptionSet<StyleColorOption
     // https://drafts.csswg.org/css-color-4/#valdef-system-color-linktext
     // Text in non-active, non-visited links. For light backgrounds, traditionally blue.
     case CSSValueLinktext:
-        return SRGBA<uint8_t> { 0, 0, 238 };
+        return useDarkAppearance ? SRGBA<uint8_t> { 158, 158, 255 } : SRGBA<uint8_t> { 0, 0, 238 };
```

Comment:
Can we change `CSSValueWebkitLink` to avoid duplication?

```
case CSSValueWebkitLink:
    if (forVisitedLink)
        return useDarkAppearance ? SRGBA<uint8_t> { 208, 173, 240 } : SRGBA<uint8_t> { 85, 26, 139 };
    return systemColor(CSSValueLinktext, options);
```

Reply:
Actually what would be even better is that we also fix `VisitedText` to be dark theme friendly and that we do this:

```
case CSSValueWebkitLink:
    if (forVisitedLink)
        return systemColor(CSSValueVisitedtext, options);
    return systemColor(CSSValueLinktext, options);
```

Reply:
Yeah no worries! Wasn't sure on the story behind WebKit link so didn't want to touch it. Likewise was going to do visited in a follow up but happy to do it here :)